packages/braid-design-system/src/lib/components/Tiles/Tiles.tsx

```diff
@@ -24,37 +25,33 @@ export const Tiles = ({
   columns = 1,
   data,
   ...restProps
-}: TilesProps) => (
-  <Box
-    className={negativeMargin('top', space)}
-    {...buildDataAttributes({ data, validateRestProps: restProps })}
-  >
+}: TilesProps) => {
+  const {
+    mobile: mobileColumns,
+    tablet: tabletColumns,
+    desktop: desktopColumns,
+    wide: wideColumns,
+  } = normalizeResponsiveValue(columns);
+
+  return (
     <Box
-      display="flex"
-      flexWrap="wrap"
-      className={negativeMargin('left', space)}
+      gap={space}
+      className={styles.tiles}
+      style={assignInlineVars({
+        ...(mobileColumns
```

Comment:
Are these values potentially `undefined`? If so, they don't need to be excluded from the object, `assignInlineVars` will remove them from the inline style, assuming we have [v2.1.0](https://github.com/vanilla-extract-css/vanilla-extract/blob/284098c02dcfb85026dab1dbe29563301095530a/packages/dynamic/CHANGELOG.md#210) or higher installed.

Reply:
Ah, [still on 2.0.3](https://github.com/seek-oss/braid-design-system/blob/0d694f136c4f0ab3c98196e81af93f7f15158855/pnpm-lock.yaml#L188-L190). Worth updating the dep to `^2.1.0`.